storage/badger/operation/seals.go

```diff
@@ -38,14 +38,30 @@ func LookupPayloadResults(blockID flow.Identifier, resultIDs *[]flow.Identifier)
 	return retrieve(makePrefix(codePayloadResults, blockID), resultIDs)
 }
 
-func IndexBlockSeal(blockID flow.Identifier, sealID flow.Identifier) func(*badger.Txn) error {
+// IndexLatestSealAtBlock persists the highest seal that was included in the fork up to (and including) blockID.
+// In most cases, it is the highest seal included in this block's payload. However, if there are no
+// seals in this block, sealID should reference the highest seal in blockID's ancestor.
+func IndexLatestSealAtBlock(blockID flow.Identifier, sealID flow.Identifier) func(*badger.Txn) error {
 	return insert(makePrefix(codeBlockToSeal, blockID), sealID)
```

Comment:
```suggestion
	return insert(makePrefix(codeBlockIDToLatestSealID, blockID), sealID)
```
Could we update the prefix name for the existing index as well?